Source/Scene/Polyline.js

```diff
@@ -285,6 +289,23 @@ define([
                     this._pickId.object.id = value;
                 }
             }
+        },
+
+        distanceDisplayCondition : {
+            get : function() {
+                return this._distanceDisplayCondition;
+            },
+            set : function(value) {
+                if (!DistanceDisplayCondition.equals(value, this._distanceDisplayCondition)) {
+                    //>>includeStart('debug', pragmas.debug);
+                    if (defined(value) && value.far <= value.near) {
```

Comment:
To avoid side effects, this check should be outside if the above `if` statement, right?